### PR TITLE
AG-13575 Fix ohlc keyboard nav

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-13575

### CODE REVIEW

**Review by alantreadway — APPROVED** [review]
LGTM

File: packages/ag-charts-community/src/chart/interaction/zoomManager.ts [hunk 1]
```diff
@@ -238,19 +238,31 @@ export class ZoomManager extends BaseManager<ZoomEvents['type'], ZoomEvents> imp
...
             return;
         }
 
-        this.state.set(callerId, newZoom);
+        state.set(callerId, newZoom);
 
-        this.axisZoomManagers.forEach((axis) => {
-            axis.updateZoom(callerId, newZoom?.[axis.getDirection()]);
-        });
+        if (this.independentAxes) {
```
Comment (lsjroberts) [inline packages/ag-charts-community/src/chart/interaction/zoomManager.ts]: I don't think this check belongs here as the `updateZoom()` fn is expected to update all axes.

**Review by lsjroberts — APPROVED** [review]
LGTM

**Comment by jacobp100** [comment]
https://github.com/ag-grid/ag-charts/pull/3169

**Pull request closed without merging**